lib/challenge.js

```diff
@@ -51,6 +82,16 @@ var pet_owner;
 var some_pet;
 
 // Answer Starts Here
+  some_pet + {
```

Comment:
There should be an assignment operator (`=`), not an addition operator (`+`) between `some_pet` and the object literal. It's more likely than not that this was a typo, though.